test/distributed/elastic/rendezvous/out_of_tree_rendezvous_test.py

```diff
@@ -0,0 +1,60 @@
+# Owner(s): ["oncall: r2p"]
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import importlib
+import pathlib
+import subprocess
+import sys
+import unittest
+
+import torch.distributed.elastic.rendezvous as rdvz
+
+
+BACKEND_NAME = "testbackend"
+TEST_PACKAGE_PATH = "/out_of_tree_test_package/"
+TEST_MODULE_PATH = TEST_PACKAGE_PATH + "src/testbackend/__init__.py"
+
+
+class OutOfTreeRendezvousTest(unittest.TestCase):
+    def test_out_of_tree_handler_loading(self):
+        current_path = str(pathlib.Path(__file__).parent.resolve())
+        rdvz._register_out_of_tree_handlers()
+        registry_dict = rdvz.rendezvous_handler_registry._registry
+
+        # test backend should not be registered as a backend
+        self.assertFalse(BACKEND_NAME in registry_dict)
+
+        # Installing test_backend package
+        subprocess.run(
```

Comment:
hm, doing an actual pip install from inside a test is not great. 

i think you could do almost the same test, but just modify the current pythonpath within the process, or find some importlib command that you pass an import path to?

Reply:
Thank you @wconstab  for the suggestion; the test became lot simpler. I have updated the tests, PTAL.